pkg/helm/charts/awslbc/awslbc.go

```diff
@@ -0,0 +1,196 @@
+// Package awslbc provides a Helm chart for installing:
+// - https://github.com/kubernetes-sigs/aws-load-balancer-controller
+// - https://github.com/aws/eks-charts/tree/master/stable/aws-load-balancer-controller
+package awslbc
+
+import (
+	"time"
+
+	"github.com/oslokommune/okctl/pkg/helm"
+)
+
+// New returns an initialised Helm chart for installing aws-alb-ingress-controller
+func New(values *Values) *helm.Chart {
+	return &helm.Chart{
+		RepositoryName: "eks",
+		RepositoryURL:  "https://aws.github.io/eks-charts",
+		ReleaseName:    "aws-load-balancer-controller",
+		Version:        "1.1.3",
+		Chart:          "aws-load-balancer-controller",
+		Namespace:      "kube-system",
+		Timeout:        5 * time.Minute, // nolint: gomnd
+		Values:         values,
+	}
+}
+
+// NewDefaultValues returns the mapped values.yml containing the default
+// values
+func NewDefaultValues(clusterName, vpcID, region string) *Values {
+	return &Values{
+		ReplicaCount:                  1, // nolint: gomnd
+		ClusterName:                   clusterName,
+		Region:                        region,
+		VpcID:                         vpcID,
+		TerminationGracePeriodSeconds: 10, // nolint: gomnd
+		EnableCertManager:             false,
```

Comment:
What is this? Could we automate creating certificates for applications?

Reply:
I believe it refers to this: https://github.com/jetstack/cert-manager, and yes, we could potentially use it now. It is essentially a service that uses lets encrypt to create certificates.

The main problem is that `AWS load balancer controller` and `cert-manager` kind of step on each others toes. With an ALB we typically terminate the encryption at the ALB and pass the request to the k8s pods in plaintext. Cert-manager attempts to terminate the encryption on the pod level. Now that we are using `AWS load balancer controller`, which supports Network Load Balancers (NLB). This might actually be possible.

☝🏽 this is based on my current understanding of the situation, without having done any further research.

Reply:
If its like how the nginx ingress controller + certbot stack works, it will terminate TLS in the ingress. In any case, its a nice improvement. More in line with the standard way of acquiring certificates with K8s. We can discuss this later